listenbrainz/mbid_mapping_writer/mbid_mapper.py

```diff
@@ -183,6 +183,9 @@ def lookup(self, artist_credit_name_p, recording_name_p):
 
             query = " ".join(cleaned_query)
 
+        if query is None or query == "":
```

Comment:
`if not query` ?

Reply:
Thats what I had, but it didn't work. I guess this is pointless, eh?